### PR TITLE
Bump graph generator lib

### PR DESCRIPTION
**Pull request by colin-grapl**
Simpy bumps us to 0.1.8

### CODE REVIEW

File: src/rust/graph-generator-lib/Cargo.toml [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+graph-descriptions = {package="grapl-graph-descriptions", version="0.2.10"}
 
 failure = "0.1.*"
 
 sha2 = "0.7.1"
 rusoto_s3 = { version="0.43.0", default_features = false, features=["rustls"] }
 rusoto_core = { version="0.43.0", default_features = false, features=["rustls"] }
 rusoto_sqs = { version="0.43.0", default_features = false, features=["rustls"] }
-grapl-config = {path="../grapl-config", version="*"}
+grapl-config = { version="0.0.1" }
```
Comment (jgrillo-grapl) [inline src/rust/graph-generator-lib/Cargo.toml:18]: We're no longer depending on the locally-built version?

File: src/rust/graph-generator-lib/Cargo.toml [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-version = "1.0.0"
+version = "0.1.8"
 authors = ["Insanitybit <insanitybit@gmail.com>"]
 edition = "2018"
+description = "Library for writing Grapl generator services"
+license = "Apache-2.0"
 
 [dependencies]
-graph-descriptions = {package="grapl-graph-descriptions", path="../graph-descriptions", version="*"}
+graph-descriptions = {package="grapl-graph-descriptions", version="0.2.10"}
```
Comment (jgrillo-grapl) [inline src/rust/graph-generator-lib/Cargo.toml:10]: Why is this version pinned now?

Comment (colin-grapl) [reply src/rust/graph-generator-lib/Cargo.toml:10]: We have to if we want to publish to crates.io

Comment (colin-grapl) [reply src/rust/graph-generator-lib/Cargo.toml:10]: I can revert? But we'd have to do this every time we publish

**Pull request merged**